Content.Shared/Interaction/Components/UnremoveableComponent.cs

```diff
@@ -1,8 +1,13 @@
+using Content.Shared.Whitelist;
 using Robust.Shared.GameStates;
 
 namespace Content.Shared.Interaction.Components
 {
     [RegisterComponent]
     [NetworkedComponent]
-    public sealed class UnremoveableComponent : Component {}
+    public sealed class UnremoveableComponent : Component 
+	{
```

Comment:
Looks likes tabs and spaces got mixed here so please fix that.

Reply:
should be fixed.